ariadne/cardano/src/Ariadne/Wallet/Cardano/WalletLayer/Kernel.hs

```diff
@@ -127,6 +129,11 @@ passiveWalletLayerCustomDBComponent logFunction keystore acidDB = do
             , pwlApplyBlocks           = liftIO . invoke . Actions.ApplyBlocks
             , pwlRollbackBlocks        = liftIO . invoke . Actions.RollbackBlocks
 
+            , pwlCreateEncryptedKey = \pp mnemonic ->
+                let seed     = mnemonicToSeedNoPassword $ unwords $ Unsafe.init mnemonic
```

Comment:
We do `Unsafe.init` here, because we append `["ariadne-v0"]` in `generateMnemonic`. It is not Cardano specific, so I propose to move it to `newWallet`. So `pwlCreateEncryptedKey` should expect mnemonic without `"ariadne-v0"`.